### PR TITLE
add ChainGovernancePrecompiled verifyAndRecord of grantCommitteeMember

### PR DESCRIPTION
**Pull request by bxq2011hust**

### CODE REVIEW

File: libprecompiled/ChainGovernancePrecompiled.cpp [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+                                  << LOG_KV("operator", userAddress) << LOG_KV("return", result);
+        } while (0);
+
+        getErrorCodeOut(callResult->mutableExecResult(), result);
+    }
+    else if (func == name2Selector[CGP_METHOD_LIST_OP])
+    {  // listOperators()
+        Address contractAddress;
+        abi.abiOut(data, contractAddress);
+        string tableName = precompiled::getContractTableName(contractAddress);
```
Comment (fisco-dev) [inline libprecompiled/ChainGovernancePrecompiled.cpp]: Codacy Issue found: [Variable 'tableName' is assigned a value that is never used.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5261917)

File: libprecompiled/ChainGovernancePrecompiled.cpp [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+                      const string& key, const string& value, const string& origin) {
+        auto entry = committeeTable->newEntry();
+        entry->setField(CGP_COMMITTEE_TABLE_VALUE, value);
+        entry->setField(CGP_COMMITTEE_TABLE_ORIGIN, origin);
+        entry->setField(CGP_COMMITTEE_TABLE_BLOCKLIMIT,
+            to_string(_context->blockInfo().number + g_BCOSConfig.c_blockLimit * 10));
+        committeeTable->insert(key, entry, make_shared<AccessOptions>(Address(), false));
+    };
+
+    auto validate = [acTable, committeeTable](
```
Comment (cyjseagull) [inline libprecompiled/ChainGovernancePrecompiled.cpp]: Recommend to split `validate`, `record` as a single function~

File: libprecompiled/ChainGovernancePrecompiled.cpp [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+        auto entries = acTable->select(SYS_ACCESS_TABLE, condition);
+        if (entries->size() == 0u)
+        {
+            PRECOMPILED_LOG(ERROR) << LOG_BADGE("ChainGovernance")
+                                   << LOG_DESC("nobody has authority of " + SYS_ACCESS_TABLE);
+            return true;
+        }
+        double total = 0;
+        map<string, double> memberWeight;
+        for (size_t i = 0; i < entries->size(); i++)
```
Comment (cyjseagull) [inline libprecompiled/ChainGovernancePrecompiled.cpp]: Recommend to add some comments.

Comment (bxq2011hust) [reply libprecompiled/ChainGovernancePrecompiled.cpp]: done

**Pull request merged**